### DISCUSSION TITLE
Heroku Server setup not working

### DISCUSSION THREAD

**Discussion by LukeRuppSr** [tonesto7/echo-speaks#1093] [Q&A]
I used Echo Speaks extensively a couple of years ago with SmartThings before Samsung decided to block it. I loved the functionality the app brings to my HA system. I bought a Hubitat C-8 Hub recently. I installed the Echo Speaks Apps and Device drivers manually in Hubitat. I could never get the Heroku Server set up working. I deleted all the Echo Speaks Apps and Drivers and then I installed the Hubitat Package Manager and installed Echo Speaks that way. Same behavior. I tried the server deployment via a browser and via my Android phone. At this point, I am at a loss of what to try next. When I create the Echo Speaks app and start the setup process, I click on the "Begin Server Setup" option, Then I am able to copy the Heroku app name and click the Deploy to Heroku button. This is where the problem starts. After clicking the deploy to Heroku button, the Heroku page opens and requests that I create a Heroku account. I click on the Heroku login button and login successfully to my Heroku account. However there are never any "Echo Speaks" headers on the page and I can create the Heroku manually but it is not connected to Echo Speaks and does not have the appropriate attributes. Is this app broken due to changes at Heroku or am I doing something wrong.? Thanks for any input you can provide.  BTW is the Echo Speaks Web Socket deprecated? It is referenced in the documentation but it is not in GitHub or in the install packages.

**Comment by LukeRuppSr** [comment]
I was able to install and configure Echo Speaks for Hubitat this evening. I made no code changes, Something must have changed on the Heroku side. In any case, thanks. I love it.

**Comment by Geospatialqandainc** [comment]
I had the EXACT same issue as @LukeRuppSr and checked this feed. I did every step the exact way I did the first time and this time it worked. 

Pleased with the system even if though it works with my 3rd gen but not working with my 4th gen echo dots right now.
